src/js/request.js

```diff
@@ -0,0 +1,19 @@
+class requestAdaptor {
+  checkStatus(response) {
+    if (response.status >= 200 && response.status < 300) {
+      return response.json();
+    }
+
+    const error = new Error(response.statusText);
+    error.response = response;
+    throw error;
+  }
+
+  get(url, callback) {
```

Comment:
Good, I like your treatment here. 

Error case is not compulsory but you handled it, well done.

Reply:
Nice helper class usage. Good job